replica/database.cc

```diff
@@ -2672,6 +2672,7 @@ future<> database::truncate_table_on_all_shards(sharded<database>& sharded_db, s
 
         return db.truncate(sys_ks.local(), cf, st, truncated_at);
     });
+    dblog.info("Truncated {}.{}", s->ks_name(), s->cf_name());
```

Comment:
This will add noise to the drop table path.
How about passing a log level from the caller so we would print it with info level only from the pure truncate path while we can have a similar message when drop is done

Reply:
@bhalevy i think drop table is an unusual operation. am i right? also, this function already has 6 parameters. i'd avoid adding new ones to the list just for the logging messages.

also, please note, we already have https://github.com/scylladb/scylladb/blob/15acceb69f401dafdbfd3e6e047e27b3c2f67046/replica/database.cc#L2453.

do you think a better alternative is to print it (and the new logging message) in `database::truncate_table_on_all_shards()` instead?

Reply:
> do you think a better alternative is to print it (and the new logging message) in database::truncate_table_on_all_shards() instead?

@tchaikov the message above is already in `truncate_table_on_all_shards`.
I initially though the message you wanted to add is per shard.
I see now that it's print once for all shards, that's reasonable indeed.
I hope it would be helpful this way...